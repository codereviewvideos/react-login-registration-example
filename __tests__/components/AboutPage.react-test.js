import React from 'react';
import AboutPage from '../../src/components/AboutPage.react';
import renderer from 'react-test-renderer';

test('Can see header', () => {
  const component = renderer.create(
    <AboutPage />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
