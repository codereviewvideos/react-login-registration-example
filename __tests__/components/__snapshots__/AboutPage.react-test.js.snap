exports[`test Can see header 1`] = `
<div>
  <h2
    className="alt-header">
    About
  </h2>
  <p>
    This example app is part of the 
    <a
      href="https://github.com/coryhouse/react-slingshot">
      React-Slingshot starter kit
    </a>
    .
  </p>
  <p>
    <a
      onClick={[Function]}
      style={Object {}}>
      Click this bad link
    </a>
     to see the 404 page.
  </p>
</div>
`;
